src/main/java/edu/wpi/grip/core/Operation.java

```diff
@@ -22,6 +22,10 @@
      */
     String getDescription();
 
+    default String getHyphenatedName(){
```

Comment:
I mean in all practicality the name and the description are purely UI elements as well. The are never actually used inside of the pipeline.